src/test/java/tests/CartTest.java

```diff
@@ -0,0 +1,37 @@
+package tests;
+
+import io.qameta.allure.Description;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import pages.CartPage;
+import pages.LandingPage;
+import pages.ProductPage;
+
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class CartTest extends BaseTest {
+
+    private ProductPage productPage;
+    private LandingPage landingPage;
+    private CartPage cartPage;
+    private String product1 = "Faded Short Sleeve T-shirts";
+    private String product2 = "Printed Chiffon Dress";
+    private String product3 = "Blouse";
+
+    @Test
+    @Description("Add Products to Cart from Product page")
+    public void addProductToCartFromProductPage() {
+        productPage = new ProductPage(driver);
```

Comment:
we can simplify that like:
landingPage = new LandingPage(driver);
productPage = landingPage.clickProductsButton();